doc/nrf/ug_thread_configuring.rst

```diff
@@ -307,16 +320,20 @@ The following table lists the supported features for each of these sets.
       - ✔
       -
 
-Replacing OpenThread Libraries
-******************************
+Replacing OpenThread libraries
+==============================
+
+Each OpenThread sample provides a possibility to replace nrfxlib's :ref:`nrfxlib:ot_libs` when configured to build OpenThread stack from source with :option:`CONFIG_OPENTHREAD_SOURCES`.
+After the sample is built, run the following command in the sample directory to overwrite the existing nrfxlib libraries with the new ones:
```

Comment:
Actually you don't need to run the command always in the sample directory. For instance you can just run:
`west build -b nrf52840dk_nrf52840 -p always nrf/samples/openthread/cli/` from root directory and there you can just run `west build -t install_openthread_libraries` without moving to sample directory.